### PR TITLE
Mh hesburgh

### PR DESCRIPTION
**Pull request by mhallwac**
Creates new spec folder, hesburghsite, and creates the config.yml and spec_helper files under that folder.  Also, adds functional folder with the func_hesburghsite_spec.rb file in it.  Adds test cases 1-9 into the func_hesburghsite_spec.rb file.

Use cases for test cases 1-9 from:
https://docs.google.com/spreadsheets/d/10yo_X20lXymyOeKu33rJBzfrbe_FdBIweVw22duCnXA/edit#gid=248186373

### CODE REVIEW

File: spec/hesburghsite/functional/func_hesburghsite_spec.rb [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      expect(page).to have_link('Story Index')
+      expect(page).to have_link('Media Gallery')
+      expect(page).to have_link('Speeches')
+      expect(page).to have_link('Further Research')
+      expect(page).to have_link('About')
+    end
+    within('.large-12.columns.contents') do
+      expect(page).to have_css('h2 a', count: 5) #should be 10 but not all header links made yet
+      expect(page).to have_css('a img', count: 10) 
+      expect(page).to have_css('li a', count: 25) #should be 57 but not all list links made yet
```
Comment (jeremyf) [inline spec/hesburghsite/functional/func_hesburghsite_spec.rb]: Are there any DOM classes for this? I can see overlap in the "h2 a", "a img" and "li a" selectors.

File: spec/hesburghsite/functional/func_hesburghsite_spec.rb [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    expect(page).to have_content('Leilani Briel')
+  end
+  scenario 'Test 9: Click on Keep in Touch tab' do
+    visit '/'
+    click_on('Keep in Touch')
+    expect(page).to have_selector('#mailinglist', visible: true)
+    within('#mc_embed_signup_scroll') do
+      expect(page).to have_css('label', count: 3)
+      #five inputs should show ensuring the 3 text boxes show
+      expect(page).to have_selector('input', count: 5)
```
Comment (jeremyf) [inline spec/hesburghsite/functional/func_hesburghsite_spec.rb]: If possible, please verify the names of the input? (e.g. 'input[name="email"]')?

File: spec/hesburghsite/functional/func_hesburghsite_spec.rb (hunk 1 above)
Comment (mhallwac) [inline spec/hesburghsite/functional/func_hesburghsite_spec.rb]: screen shot 2017-05-24 at 10 37 13 am

I cannot find any DOM classes to make it more specific.

Comment (jeremyf) [inline spec/hesburghsite/functional/func_hesburghsite_spec.rb]: Thanks for investigating; And :( for us

**Pull request merged**